app/src/main/res/drawable/filled_star_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="12dp" android:tint="#FF3365"
```

Comment:
remove this one and use tint in image view

Reply:
Done